sycl/include/CL/sycl/backend.hpp

```diff
@@ -131,5 +143,29 @@ make_buffer(const typename backend_traits<Backend>::template input_type<
   return buffer<T, Dimensions, AllocatorT>(
       reinterpret_cast<cl_mem>(BackendObject), TargetContext, AvailableEvent);
 }
+
+template <backend Backend>
+typename std::enable_if<
+    detail::InteropFeatureSupportMap<Backend>::MakeKernel == true, kernel>::type
+make_kernel(const typename backend_traits<Backend>::template input_type<kernel>
```

Comment:
Do we really need/want to add "generic" make-functions, and not have every backend add their own?

Reply:
> For each SYCL runtime class T which supports SYCL application interoperability, a specialization of the appropriate template function make_{sycl_class} where {sycl_class} is the class name of T, must be defined, which takes a SYCL application interoperability native backend object and constructs and returns an instance of T.

To me this suggest that we should not define the default ones, as specializations "**must be defined**".

Reply:
I am OK to have it this way to avoid code duplication.